### PR TITLE
Updated ReadME for Rust branch

### PR DESCRIPTION
**Pull request by Roee-87**
updated ReadME.  Using the PR for a side by side comparison.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,201 +7,168 @@
...
-So that we can leave Spice.ai running, add the quickstart pod in a new terminal tab or window. If you are running in GitHub Codespaces, you can open a new terminal by clicking the split-terminal button in VS Code.
+You can enter custom queries:
 
 ```bash
-spice add quickstarts/serverops
+sql> SELECT topics FROM eth_recent_logs LIMIT 5;
 ```
 
-The Spice.ai CLI will download the ServerOps quickstart pod and add the pod manifest to your project at `spicepods/serverops.yaml`.
+### Run through a data demo step by step here. Include a video showing the steps
```
Comment (Roee-87) [inline README.md:124]: Transition to Samples

Comment (Roee-87) [reply README.md:124]: One sample is sufficient.

Enumerate 3 other quick start tutorials:
spice.ai login, local dataset configure, ML models.

Comment (Roee-87) [reply README.md:124]: Include one of the streaming datasets.

Comment (phillipleblanc) [reply README.md:124]: Include DuckDB and Postgres backends

File: README.md [hunk 2]
```diff
@@ -7,201 +7,168 @@
...
+⚠️ The vision to make it easy to build intelligent applications that learn is a vast undertaking. We haven't figured it all out or solved all the problems yet, so we’re inviting you on this journey and are looking for feedback the direction.
 
-### Run the ServerOps application
+The team at SpiceAI is committed to creating a developer community
 
-To see how Spice.ai makes creating intelligent applications easy, try running and reviewing the sample ServerOps Node or Powershell apps, `serverops.js` and `serverops.ps1`.
+Spice.ai and spicerack.org are both pre-release, early, alpha software. Until v1.0, Spice.ai may have gaps, including limited deep learning algorithms, training-at-scale, and simulated environments..
 
-Node:
+Our intention with this preview is to work with developers early to define and create the developer experience together. 🚀 See the [Roadmap to v1.0-stable](https://github.com/spiceai/spiceai/blob/rust/docs/RELEASE.md) for upcoming features.
```
Comment (phillipleblanc) [inline README.md]: Remove

File: README.md [hunk 3]
```diff
@@ -7,201 +7,168 @@
...
-Powershell:
+We greatly appreciate and value your support! You can help Spice.ai in a number of ways:
 
-```ps
-./serverops.ps1
-```
+- ⭐️ Star this repo.
+- Build an app with Spice.ai and send us feedback and suggestions at [hey@spice.ai](mailto:hey@spice.ai) or on [Discord](https://discord.gg/kZnTfneP5u).
+- [File an issue](https://github.com/spiceai/spiceai/issues/new) if you see something not quite working correctly.
+- Follow us on [Reddit](https://www.reddit.com/r/spiceai), [Twitter](https://twitter.com/SpiceAIHQ), and [LinkedIn](https://www.linkedin.com/company/74148478).
```
Comment (phillipleblanc) [inline README.md]: Update `Twitter -> X`, remove reddit

File: README.md [hunk 4]
```diff
@@ -7,201 +7,168 @@
...
+- Join our team ([We’re hiring!](https://spice.ai/careers))
+- Contribute code or documentation to the project (see [CONTRIBUTING.md](CONTRIBUTING.md)).
 
-### Next steps
+We’re also starting a community call series soon!
 
-Congratulations! In just a few minutes you downloaded and installed the Spice.ai CLI and runtime, created your first Spicepod, trained it, and got a recommendation from it.
+Thank you for sharing this journey with us.
```
Comment (phillipleblanc) [inline README.md]: Remove this, add back `Next steps`

**Comment by Roee-87** [comment]
This PR updates the ReadME for the Rust branch in preparation for the merge to main.  
Other changes include:
-deleting release ntoes
-deleting principles.md
-removing contents from Roadmap.md

In future, I'll submit separate PR's for each of these issues.

File: CONTRIBUTING.md
Comment (phillipleblanc) [inline CONTRIBUTING.md:1]: Why is this file deleted?

File: docs/PRINCIPLES.md
Comment (phillipleblanc) [inline docs/PRINCIPLES.md:1]: Why is this file deleted?

File: docs/README.md
Comment (phillipleblanc) [inline docs/README.md:1]: Why this change?

File: docs/ROADMAP.md
Comment (phillipleblanc) [inline docs/ROADMAP.md:1]: We said we wanted to keep the ROADMAP - but just have a Coming Soon! placeholder

File: docs/release_notes/v0.1.0-alpha.2.md
Comment (phillipleblanc) [inline docs/release_notes/v0.1.0-alpha.2.md:1]: I don't think we should delete the old release notes as part of this release

File: spiceai_try_for_free.png
Comment (phillipleblanc) [inline spiceai_try_for_free.png:1]: Why is this here?

File: README.md [hunk 5]
```diff
@@ -7,211 +7,336 @@
...
----
+**Spice.ai** is an open source runtime environment that enables developers to leverage rapid access to time-series data. The Spice AI Runtime enables the best in class high performance queries for powering an array of data driven applications.
 
 ⚠️ **DEVELOPER PREVIEW ONLY** Spice.ai is under active **alpha** stage development and is not intended to be used in production until its **1.0-stable** release.
 
----
+## Quick Start
 
-📢 Read the Spice.ai announcement blog post at [blog.spiceai.org](https://blog.spiceai.org).
+SpiceAI uses Go and Rust. Be sure to have both of these installed before proceeding.
```
Comment (phillipleblanc) [inline README.md]: They don't need to have Go or Rust installed - the binaries are pre-compiled.

File: README.md [hunk 6]
```diff
@@ -7,211 +7,336 @@
...
 
-### Community-Driven Data Components
+This creates a spicepod.yaml file in your directory.
 
-Spice.ai also includes a library of community-driven [data components](https://github.com/spiceai/data-components-contrib) for streaming and processing time series data, enabling developers to quickly and easily combine data with learning to create intelligent models.
+```yaml
+version: v1beta1
+kind: Spicepod
+name: my_spiceai_project
+```
```
Comment (phillipleblanc) [inline README.md]: I don't think its necessary to show the yaml - this is valuable screen real estate (near the top of the README), let's not waste it.

File: README.md [hunk 7]
```diff
@@ -7,211 +7,336 @@
...
 
-Spice.ai takes a developer-first approach, and is focused on a fast, iterative, inner development loop, enabling developers to get started with ML in minutes instead of months.
-
----
+**Spice.ai** is an open source runtime environment that enables developers to leverage rapid access to time-series data. The Spice AI Runtime enables the best in class high performance queries for powering an array of data driven applications.
 
 ⚠️ **DEVELOPER PREVIEW ONLY** Spice.ai is under active **alpha** stage development and is not intended to be used in production until its **1.0-stable** release.
 
----
+## Quick Start
```
Comment (phillipleblanc) [inline README.md]: Before we go into the quickstart, we need the use cases here to get people excited to try it out.

File: README.md [hunk 8]
```diff
@@ -7,211 +7,336 @@
...
 
-Now imagine learning Python or R, neural networks, deep-learning algorithms and building a system that streams and processes time-series data to do that. With Spice.ai — which includes a time-series ML engine accessible over HTTP APIs, a library of community-driven components for data streaming and processing, and an ecosystem of pre-created ML configurations — you can build upon the experience of the community instead of doing it all yourself. You can focus on business logic and building your application instead of the ML.
+```bash
+spice add spiceai/quickstart
+```
 
-### Pre-release software
+You should see the runtime updated with the new datasets.
+Also, the spicepod.yaml file will be updated with a new dependency that
+references the Lukekim/demo spicepod.
```
Comment (phillipleblanc) [inline README.md]:
```suggestion
references the spiceai/quickstart spicepod.
```

File: README.md [hunk 9]
```diff
@@ -7,211 +7,336 @@
...
 
 ```bash
-cd $HOME
-git clone https://github.com/spiceai/quickstarts
-cd quickstarts/serverops
+Locally accelerate this dataset (y/n)? y
 ```
 
-Step 2. Start the Spice runtime with `spice run`:
+You should see the following outpout from your runtime terminal:
```
Comment (phillipleblanc) [inline README.md]:
```suggestion
You should see the following output from your runtime terminal:
```

File: README.md [hunk 10]
```diff
@@ -7,211 +7,336 @@
...
 
 ```bash
-cd $HOME/quickstarts/serverops
-spice run
+2024-02-21T22:49:10.038461Z  INFO runtime: Loaded dataset: eth_recent_blocks
 ```
 
-Step. 3. In a new terminal, add the ServerOps quickstart pod:
+**Step 4.** In a new terminal window, use the SpiceAI SQL RPL to query the dataset
```
Comment (phillipleblanc) [inline README.md]:
```suggestion
**Step 4.** In a new terminal window, use the SpiceAI SQL REPL to query the dataset
```

Comment (phillipleblanc) [reply README.md]: But I don't think we want to call it `REPL` in the README

File: docs/release_notes/v0.1.0-alpha.2.md
Comment (Roee-87) [reply docs/release_notes/v0.1.0-alpha.2.md:1]: I discussed this with Aurash yesterday...since this is a revamp of the old OSS project and essentially a v0.1, there is no link between the previous versions and this one.

File: CONTRIBUTING.md
Comment (Roee-87) [reply CONTRIBUTING.md:1]: Error on my part.  
It's now replaced with the updated version. (removed CONTRIBUTION.rust.md and replaced with updated CONTRIBUTION.md).

File: docs/release_notes/v0.1.0-alpha.2.md
Comment (Roee-87) [reply docs/release_notes/v0.1.0-alpha.2.md:1]: Reinsterted

File: README.md (hunk 6 above)
Comment (Roee-87) [reply README.md]: removed.

File: README.md (hunk 5 above)
Comment (Roee-87) [reply README.md]: Removed.

File: docs/ROADMAP.md
Comment (Roee-87) [reply docs/ROADMAP.md:1]: Added placeholder.

File: docs/PRINCIPLES.md
Comment (Roee-87) [reply docs/PRINCIPLES.md:1]: Reinserted with Rust branch version.

File: docs/README.md
Comment (Roee-87) [reply docs/README.md:1]: Reverted.  Reinserted PRINCIPLES.md and updated docs/README.md with link

File: spiceai_try_for_free.png
Comment (Roee-87) [reply spiceai_try_for_free.png:1]: removed.

File: README.md (hunk 7 above)
Comment (Roee-87) [reply README.md]: added
